.gitignore

```diff
@@ -1,4 +1,5 @@
 env
 .vscode
 __pycache__
-venv
\ No newline at end of file
+venv
+.DS_Store
```

Comment:
Added this `.DS_STORE` because I'm working on mac and these files get automatically generated and they don't need to be committed.